External/Plugins/ProjectManager/Projects/Project.cs

```diff
@@ -76,6 +75,7 @@ public Project(string path, CompilerOptions compilerOptions)
         }
 
         public abstract string Language { get; }
+        public abstract string LanguageDisplayString { get; }
```

Comment:
I guess adding an `abstract` property here is a breaking API change.. Are there any third-party plugins that depend on this API?